Library/Formula/bro.rb

```diff
@@ -22,6 +22,12 @@ class Bro < Formula
   depends_on "geoip" => :recommended
   depends_on "openssl"
 
+  # Initially empty directories
+  skip_clean "spool"
+  skip_clean "logs"
```

Comment:
Bro doesn't create a var/ directory anywhere by default, just $prefix +

    bin/     etc/     include/ lib/     logs/    share/   spool/

I suppose it does make sense for logs/ and spool/ to be in var/, but the fix for that is a little more involved.